### PR TITLE
Move scheduling actions from the Action to the View

### PR DESCRIPTION
**Pull request by rudigiesler**

### CODE REVIEW

**Comment by rudigiesler** [comment]
Ready for review.

File: go/conversation/view_definition.py [hunk 1]
```diff
@@ -574,11 +577,32 @@ def post(self, request, conversation):
...
+                self.action.action_display_name,))
+        else:
+            self.schedule_action(action_data, conversation)
+            messages.info(request, 'Action successfully scheduled: %s!' % (
+                self.action.action_display_name,))
         return self._action_done(request, conversation)
 
+    def schedule_action(self, action_data, conversation):
+        scheduled_for = action_data.pop('scheduled_datetime')
+        action_data.pop('display')
```
Comment (hodgestar) [inline go/conversation/view_definition.py:594]: What is `action_data['display']`?

**Comment by hodgestar** [comment]
One question, but otherwise looks good!

File: go/conversation/view_definition.py (hunk 1 above)
Comment (rudigiesler) [inline go/conversation/view_definition.py:594]: It comes from here: https://github.com/praekelt/vumi-go/pull/1364/files#diff-a6cac4bf3e6adc22b863f06ebdc14232R565 . I have no idea what it's for though.

Comment (hodgestar) [inline go/conversation/view_definition.py:594]: Ah, it's for displaying the details of the action to the user during action confirmation. That means that ignoring it here is the correct thing to do.

**Comment by hodgestar** [comment]
:+1:

**Pull request merged**